Rename Starlite (#1453)
starlite-api/starlite: docs/release-notes/changelog.rst

        :pr: 1277

        Add a ``__version__`` constant to the ``starlite`` namespace, containing a

>         ``starlitetyping.NamedTuple``, holding information about the currently

Reviewer: What happened here? Regex gone wild? :wink: 
```suggestion
        :class:`NamedTuple <typing.NamedTuple>`, holding information about the currently
```

        Add a new ``version`` command to the CLI which displays the currently installed
        version of Starlite

    .. change:: Enhance CLI autodiscovery logic
        :type: feature
        :breaking:
        :pr: 1322

        Update the CLI :ref:`usage/cli:autodiscovery` to only consider canonical modules app and application, but every
        ``starlite.app.Starlite`` instance or application factory able to return a ``Starlite`` instance within
        those or one of their submodules, giving priority to the canonical names app and application for application
        objects and submodules containing them.

        .. seealso::

>             :ref``starliteusage/cli:autodiscovery``

Reviewer: ```suggestion
            :ref:`CLI autodiscovery <usage/cli:autodiscovery>`
```

    .. change:: Remove ``picologging`` dependency from the ``standard`` package extra
        :type: misc
        :breaking:

>         :pr: 131

Reviewer: ```suggestion
        :pr: 1313
```

        :ref:`usage/responses:Response Headers` and :ref:`usage/responses:Response Cookies` now have the same
        interface, along with the ``headers`` and ``cookies`` keyword arguments to

>         ``starlite.response.Response``. They each allow to pass either a
>         ``Mapping[st``starlitetyping.Mapping``, e.g. a dictionary, or a ``starlitetyping.Sequence`` of

Reviewer: ```suggestion
        ``starlite.response.Response``. They each allow to pass either a
        `:class:`Mapping[str, str] <typing.Mapping>`, e.g. a dictionary, or a :class:`Sequence <typing.Sequence>` of
```

        The ``starlite`` namespace has been cleared up, removing many names from it, which now have to be imported from
        their respective submodules individually. This was both done to improve developer experience as well as reduce
        the time it takes to ``import starlite``.

    .. change:: Fix resolving of relative paths in ``StaticFilesConfig``
        :type: bugfix
        :pr: 1256

>         Using a relative ``pathlib.Path` did not resolve correctly and result in a ``NotFoundException``

Reviewer: ```suggestion
        Using a relative :class:`pathlib.Path` did not resolve correctly and result in a ``NotFoundException``
```
